CONTRIBUTING.md

```diff
@@ -22,20 +22,18 @@ For contributors who want to work up pull requests, the workflow is roughly:
 1. Create a topic branch from where you want to base your work (usually master).
 2. Make commits of logical units.
 3. Make sure your commit messages are in the proper format (see [below](#commit-message-format)).
-4. If necessary, you may use `make update-codegen` to update generated code.
-5. Make sure the tests pass (`go test ./...`), and add any new tests as appropriate.
+4. Make sure your [code builds][build-cvo] and follows the [coding style](#coding-style)
```

Comment:
Any reason we not suggesting `make update-codegen` ?

Reply:
Do we expect that users will notice the test failures in the PR and then run it?

Reply:
update-codegen.sh is a tombstone and does nothing: https://github.com/openshift/cluster-version-operator/blob/master/hack/update-codegen.sh

@wking [suggested](https://github.com/openshift/cluster-version-operator/pull/871#discussion_r1047353017) that I fix some style in the related makefile targets so I ended up removing all that to avoid maintenance of code with no value

Reply:
Leaving a note here for our future selves after a Slack convo. The answer is that the code generation happens in openshift/api repository and then we vendor it.